doc-Installing_on_Microsoft_Azure/topics/installation.adoc

```diff
@@ -31,32 +30,130 @@ ifdef::cfme[]
 endif::cfme[]
 
 [[uploading-the-appliance-to-microsoft-azure]]
-=== Uploading the Appliance to Microsoft Azure
+=== Uploading the {product-title} Virtual Appliance to Microsoft Azure
+
+You can upload the appliance to an Azure environment using the following two methods. 
 
-In order to upload the {product-title} appliance file to Microsoft Azure, ensure the following requirements are met:
+* Using Azure PowerShell script
+* Using Azure Command-Line Interface (Azure CLI)
 
-* Approximately 2 GB of space for each VHD image; 44+ GB of space, 8 GB RAM, and 4 VCPUs for the {product-title} appliance.
+To upload the {product-title} appliance file to Microsoft Azure, ensure the following requirements are met:
+
+* Approximately 2 GB of space for each VHD image; 44+ GB of space, 16 GB RAM, and 4 VCPUs for the {product-title} appliance.
+* link:https://azure.microsoft.com/en-us/free/[Microsoft Azure Account]. 
 * Administrator access to the Azure portal.
+* xref:converting-aligning-cfme-appliance-image[Converting and Aligning the {product-title} Virtual Appliance Image]
 * Depending on your infrastructure, allow time for the upload.
 
+
+[IMPORTANT]
+====
+Azure requires that the uploaded Virtual Hard Disk (VHD) files are in a fixed format. The {product-title_short} virtual appliance image .vhd file is dynamic by default. Currently, the Azure Powershell script and Azure CLI does not automatically convert the dynamic .vhd file to fixed during upload. To upload using either method, the {product-title_short} virtual appliance image .vhd file must be first converted from dynamic to fixed, and properly aligned to the nearest 1 MB boundary. Once converted and properly aligned, you can then upload the appliance virtual image .vhd file using either the Azure PowerShell or Azure CLI method.
+====
+
+
+[[converting-aligning-cfme-appliance-image]]
+==== Converting and Aligning the {product-title_short} Virtual Appliance Image
+
+Complete the following procedure to ensure the {product-title} dynamic .vhd file is aligned correctly, and is in a fixed-size VHD format. 
+
 [NOTE]
 ====
-These are the procedural steps as of the time of writing. For more information, consult the following Azure documentation. 
+All .vhd files provisioned for Microsoft Azure must be aligned to the nearest 1 MB boundary and be in a fixed-size VHD format. If you attempt to upload a dynamic .vhd file or one that is not properly aligned, virtual appliance provisioning will not work.
```

Comment:
Good point. In fact, I remember deciding to remove it after adding that info in the Important box, but clearly did not. Yeah, certainly feels redundant - removed it now.